src/main/java/seedu/address/logic/commands/UnEnrollCommand.java

```diff
@@ -68,16 +68,12 @@ public CommandResult execute(Model model) throws CommandException {
         Tutorial tutorial = optionalTutorial.get();
 
         Participation p = new seedu.address.model.participation.Participation(student, tutorial, new ArrayList<>());
```

Comment:
Should this be participation instead? Is there any reason the other constructor is not used instead?